src/hotel/swagger.yaml

```diff
@@ -0,0 +1,97 @@
+swagger: '2.0'
+info:
+  description: >-
+    This is a microservice that handles hotel data for Bee Travel
+  version: 1.0.0
+  title: Bee Travel Hotel Data Service
+  contact:
+    email: moficodes@gmail.com
+  license:
+    name: Apache 2.0
+    url: 'https://github.com/bee-travels/bee-travels-node/blob/master/LICENSE'
+host: 'localhost:9002'
+basePath: /api/v1/hotels
+tags:
+  - name: Hotels
+    description: Operations associated with hotel data
+schemes:
+  - http
+paths:
+  /hotels/{city}/{country}:
```

Comment:
The `basePath` on line 13 has a `/hotels` in it already so the `/hotels` isn't needed here
ex. `http://localhost:9002/api/v1/hotels/hotels/Los%20Angeles/United%20States`